diff --git a/src/niri.rs b/src/niri.rs
index 88dbd0ee..48b2b093 100644
--- a/src/niri.rs
+++ b/src/niri.rs
@@ -4470,7 +4470,7 @@ impl Niri {
                     size,
                     scale,
                     Transform::Normal,
-                    Fourcc::Abgr8888,
+                    Fourcc::Argb8888,
                     elements,
                 );
                 if let Err(err) = &res {
diff --git a/src/ui/screenshot_ui.rs b/src/ui/screenshot_ui.rs
index 4f2dba3e..aece1ff2 100644
--- a/src/ui/screenshot_ui.rs
+++ b/src/ui/screenshot_ui.rs
@@ -462,7 +462,7 @@ impl ScreenshotUi {
             .to_buffer(1, Transform::Normal, &Size::from((1, 1)));
 
         let mapping = renderer
-            .copy_texture(&texture, buf_rect, Fourcc::Abgr8888)
+            .copy_texture(&texture, buf_rect, Fourcc::Argb8888)
             .context("error copying texture")?;
         let copy = renderer
             .map_texture(&mapping)
